### DISCUSSION TITLE
Template installation

### DISCUSSION THREAD

**Discussion by jpogran** [puppetlabs-toy-chest/pct#113] [Feedback]
We are planning to have Puppet Content Templates be installed from the Puppet Forge. This is a place to discuss the various options around this, including whether to support a 'local' or 'git' installation option.

**Comment by jstraw** [comment]
It would be extremely helpful for my organization to have an option in a global config to point at a local folder or git repository that contained templates specific to an internal organization, or otherwise.  While an artifactory api path would work for that purpose, it would force a change of method from our current r10k/puppetfiles and git design we've been using for years.

Maybe instead of a global config, it could be that pct could add to the metadata.json that it needs a git or forge pct module, so pct would see that and ensure it was placed in cache before doing anything.

**Reply by sanfrancrisko** [reply]
As of `0.4.0` it is now possible to install templates from any remote HTTP/S resource - [see here](https://github.com/puppetlabs/pdkgo/blob/609aaf40b6a662381cd5d67a68ca4e1de4ef8192/README.md#installing-template-packages)
